plugin-template

```diff
@@ -68,6 +68,7 @@ DEFAULT_SETTINGS = {
     "test_deprecations": True,
     "test_performance": False,
     "test_released_plugin_with_next_pulpcore_release": False,
+    "test_reroute": True,
```

Comment:
What was the motivation for making it optional?

Reply:
https://github.com/pulp/pulpcore/runs/5379632837?check_suite_focus=true#step:8:511

Reply:
Make sense, thanks!